tests/run-staging/quote-nested-4.scala

```diff
@@ -5,7 +5,7 @@ object Test {
   given Toolbox = Toolbox.make(getClass.getClassLoader)
   def main(args: Array[String]): Unit = withQuoteContext {
 
-    val q = '{
+    val q = '{ (qtx: QuoteContext) ?=>
```

Comment:
```suggestion
    val q = '{ (using qctx: QuoteContext) =>
```

Reply:
Can we remove the explicit implicit function, and make eta-expansion do the job?

Reply:
We can if there is enough information in the surrounding context. I chose his one because it is the simplest change.